nip47/event_handler.go

```diff
@@ -75,7 +76,7 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 				Message: fmt.Sprintf("Failed to save nostr event: %s", err.Error()),
 			},
 		}
-		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss)
+		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss, svc.keys.GetNostrSecretKey())
```

Comment:
there is no need to pass this as an argument, is there? since it's already available in the `svc` and we're calling a method on the `svc`

Reply:
Ah, I see it depends if the app has its own key or not (it's an old app)

Reply:
@frnandu  I think this is wrong. Actually I don't think we should reply in this case because we don't even know what key to use now